CMakeLists.txt

```diff
@@ -1,25 +1,85 @@
-
 cmake_minimum_required(VERSION 3.23)
 project(beman LANGUAGES CXX)
 
 include(CTest)
-
 include(FetchContent)
-FetchContent_Declare(
-  example
-  GIT_REPOSITORY https://github.com/beman-org/example.git
-  GIT_TAG        main
-)
 
-FetchContent_Declare(
-  scn
-  GIT_REPOSITORY https://github.com/eliaskosunen/scnlib
-  GIT_TAG        v2.0.2
+option(
+  BEMAN_USE_MAIN_BRANCHES
```

Comment:
@tzlaine I made up this CMake option for users that want a "develop" workflow. I picked a name that made sense to me, but it could be anything. But the user workflow is to set `BEMAN_USE_MAIN_BRANCHES` and you'll get whatever the default branch is per library, as defined in `libraries.json` in this project.